test/extensions/filters/http/common/fuzz/uber_filter.cc

```diff
@@ -94,6 +111,32 @@ void UberFilterFuzzer::fuzz(
   reset();
 }
 
+void UberFilterFuzzer::guideAnyProtoType(test::fuzz::HttpData* mutable_data, int seed) {
```

Comment:
Could the definition of this function be moved to `uber_per_filter.cc`? There was some re-org here, but all the filter-specific code is there.

Reply:
Done